README.md

```diff
@@ -167,6 +212,22 @@ allowedTags: false,
 allowedAttributes: false
 ```
 
+#### "What if I want to allow empty attributes?"
```

Comment:
This implies they are not allowed by default. They are allowed, just only where it makes sense.

This could be a better heading:

"What if I want to allow empty attributes, even for cases like `href` that normally don't make sense?"

Reply:
Updated